main.js

```diff
@@ -44,5 +44,20 @@ const toggleFooterMenu = (e) => {
 };
 
 toggleFooterMenu(mediaQueryList);
+mediaQueryList.addEventListener("change", toggleFooterMenu);
 
-mediaQueryList.addEventListener("change", toggleFooterMenu);
\ No newline at end of file
+let scrollButton = document.getElementById("scrollBtn");
+window.onscroll = function () { scrollFunction() };
+scrollButton.style.display = "none";
+
+function scrollFunction() {
+  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+    scrollButton.style.display = "block";
+  } else {
+    scrollButton.style.display = "none";
+  }
+}
```

Comment:
you might wanna adjust position of the button - currently even if the screen is wide, the button floats over the container. I think it would be better if it was closer to the right side of the screen.